sgit/enums.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum
+
+
+class OrderAlgorithms(Enum):
+    ALPHABETICAL = 10
+    TIME = 20
+    SEMVER = 30
+
+
+class SelectionMethods(Enum):
+    SEMVER = 10
+    EXACT = 20
```

Comment:
Maybe use `LITERAL` instead of `EXACT`? You decide.

Reply:
I might, but let me think about it and i will push it in a future commit if i change it. I find both options a little bit wonky tbh

Reply:
I added in this as a idea card in the 1.0.0 project for keeping track of this suggestion